model/constants.py

```diff
@@ -85,6 +88,9 @@ def get_empty_changes():
             'genes': [],
             'reactions': [],
         },
+        'changed': {
```

Comment:
there are already "measured" reactions, maybe this key should be reused and new `changed` key is not necessary